docs/pages/versions/unversioned/sdk/apple-authentication.md

```diff
@@ -3,14 +3,14 @@ title: AppleAuthentication
 sourceCodeUrl: "https://github.com/expo/expo/tree/sdk-36/packages/expo-apple-authentication"
 ---
 
-This library provides Apple authentication for iOS standalone apps in the managed and bare workflows. Beginning with iOS 13, any app that includes third-party authentication options **must** provide Apple authentication as an option in order to comply with App Store Review guidelines. Learn more about Apple authentication on the ["Sign In with Apple" website](https://developer.apple.com/sign-in-with-apple/).
+This library provides Apple authentication for iOS. It does not yet support Android or web.
```

Comment:
```suggestion
This library provides Apple authentication for iOS 13+. It does not yet support lower iOS versions, Android, or web.
```

@brentvatne based on my understanding this is correct, and also helps clarify in testing why it might return unavailable? I test on iOS9 all the time still so I would be confused (assuming my understanding is correct)